concepts/05 UI Components/DataGrid/07 Choose a Data Management Component/15 PivotGrid.md

```diff
@@ -0,0 +1,11 @@
+Displays and analyzes multidimensional data. Used for OLAP reporting and analytics. Supports custom aggregation and pivoting.
+
+<img src="/images/pivotgrid/pivot-grid-choose.png" alt="DevExtreme PivotGrid" />
+
+Use it for:
+
+- Sales analysis by region, period, or product  
+- Comparison of financial metrics  
+- Pivot tables for reports and charts  
```

Comment:
I wanted to promo charts integration feature
Maybe smth like 'presenting information in tabular form and as charts'